### PR TITLE
Render error messages from validation in deletion model

### PR DESCRIPTION
**Pull request by shlok007**

### CODE REVIEW

File: app/models/deletion.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Deletion < ActiveRecord::Base
   private
 
   def version_is_indexed
-    errors.add(:number, "is already deleted") unless @version.indexed?
+    errors.add(:base, "#{rubygem_name} #{version} is already deleted") unless @version.indexed?
```
Comment (indirect) [inline app/models/deletion.rb]: Looking good! Let's make this match the old message: "has already been deleted" 👍

Comment (shlok007) [inline app/models/deletion.rb]: Sure. :+1:

**Comment by dwradcliffe** [comment]
LGTM

Is there a test for this case?

**Comment by shlok007** [comment]
I was able to find the a test in https://github.com/rubygems/rubygems.org/blob/master/test/functional/api/v1/deletions_controller_test.rb#L112 . Do let me know if more tests are to be added for this change. :)

**Comment by dwradcliffe** [comment]
Ah, good find. 👍🏻

**Comment by indirect** [comment]
@homu r+

**Pull request merged**